website_sale_browse_only/__manifest__.py

```diff
@@ -0,0 +1,15 @@
+{
+    "name": "Website Sale Browse Only",
+    "version": "16.0.1.0.0",
+    "category": "Website",
+    "author": "Coop IT Easy SC, Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/e-commerce",
+    "license": "AGPL-3",
+    "depends": ["website_sale_stock"],
+    "data": [
+        "views/website.xml",
+        "views/res_config_settings.xml",
+        "views/website_sale_template.xml",
+    ],
+    "installable": True,
+}
```

Comment:
the `summary` key is missing (and the `installable` key can be removed, as it’s as its default value).